src/clj/rems/form_validation.clj

```diff
@@ -62,7 +62,7 @@
       ;; Schema validation guarantees that it's either a s/Str or
       ;; a [[{:column s/Str :value s/Str}]], and we've ruled out s/Str
       ;; in wrong-value-type-error
-      (when (not (every? row-ok? (:field/value field)))
+      (when (not (every? row-ok? value))
```

Comment:
How about using `when-not` or inverting `row-ok?` to `invalid-row?` and using `some`?

Reply:
I prefer pushing the negations as "out" as possible so that you don't need to keep tracking if something is positive or negative while reading. So I prefer `(every? row-ok? value)`. I'll switch to a `when-not` though.

Reply:
Well in this case "some invalid" reads to me as "positive" all the way through both are fine for me :+1: .